packages/nuxt/src/app/plugins/chunk-reload.client.ts

```diff
@@ -0,0 +1,23 @@
+import { defineNuxtPlugin } from '#app/nuxt'
+import { useRouter } from '#app/composables/router'
+
+export default defineNuxtPlugin((nuxtApp) => {
+  const router = useRouter()
+
+  let chunkError: TypeError | null = null
+  nuxtApp.hook('app:chunkError', ({ error }) => { chunkError = error })
```

Comment:
It might be nice if we can keep all chunk errors (otherwise only latest is available, loosing errors)